pkg/scheduler/tests/scheduler_perf_test.go

```diff
@@ -19,12 +19,19 @@ package tests
 import (
     "github.com/cloudera/yunikorn-core/pkg/common/configs"
     "github.com/cloudera/yunikorn-core/pkg/entrypoint"
+    "github.com/cloudera/yunikorn-core/pkg/log"
     "github.com/cloudera/yunikorn-scheduler-interface/lib/go/si"
+    "go.uber.org/zap"
+    "strconv"
     "testing"
     "time"
 )
 
 func TestSchedulerThroughput5KNodes(t *testing.T) {
+    if testing.Short() {
+        t.Skip("skipping performance testing in short mode")
+    }
+    log.InitAndSetLevel(zap.WarnLevel)
```

Comment:
When we mark this as a benchmark and correctly implement the test we should not need it.
As a temporary workaround I can understand that moving to the standard level (Info) would be more in line to get a view of how the scheduler behaves. Unless the standard is Warn for most deployments.

Reply:
Make sense to me, thanks.